packages/components/CollectionView.tsx

```diff
@@ -45,11 +45,12 @@ export const CollectionView: React.FC<{
       }}
       disabled={item.id === "" && item.twitterUrl === ""}
     >
-      <LegacyTertiaryBox
-        noBrokenCorners={size === "XS"}
-        mainContainerStyle={sizedStyles.boxMainContainer}
-        width={sizedStyles.box.width}
-        height={sizedStyles.box.height}
+      <TertiaryBox
+        style={{
```

Comment:
For the style we usually avoid `...styles.blabla` in the style object so for that you can do like that:
```
style={[sizedStyles.boxMainContainer, { width: sizedStyles.box.width, height: sizedStyles.box.height }] }
```